tests/root_gauge_factory_proxy/test_root_gauge_factory_proxy_admin.py

```diff
@@ -0,0 +1,275 @@
+    assert root_gauge_factory.future_owner() == root_gauge_factory_proxy
+    assert root_gauge_factory.owner() == alice  # alice is still the owner
+
+    # ---
+    # accept transfer ownership: only proxy (root_gauge_factory.future_owner) can accept
+    # this is callable by anyone through the proxy
+
+    # unauthorised accounts cannot complete ownership transfer through the factory
+    for acct in [default_ownership_admin, default_emergency_admin, alice, bob, charlie]:
+        with brownie.reverts():
+            root_gauge_factory.accept_transfer_ownership({"from": acct})
+
+    # completing transfer can be only be done through the root gauge factory proxy now
+    for acct in [alice, bob, charlie]:
+        root_gauge_factory_proxy.accept_transfer_ownership(root_gauge_factory, {"from": acct})
+        assert root_gauge_factory.owner() == root_gauge_factory_proxy
+        chain.undo()
+
+
+def test_set_killed(
+    root_gauge,
+    root_gauge_factory,
+    root_gauge_factory_proxy,
+    alice,
+    bob,
+    charlie,
+    chain,
+):
+
+    default_ownership_admin = root_gauge_factory_proxy.ownership_admin()
+    default_emergency_admin = root_gauge_factory_proxy.emergency_admin()
+
+    # alice deployed the factory and set herself as the owner of the factory contract
+    assert root_gauge_factory.owner() == alice
+
+    # alice is the only one who can kill gauges
+    root_gauge.set_killed(True, {"from": alice})
+    assert root_gauge.is_killed()
+    assert root_gauge.inflation_params()[0] == 0  # inflation rate of root gauge should be 0
+    chain.undo()
+
+    # proxy cannot kill gauges yet, and neither can alice do it through the proxy:
+    for unauthorised_admins in [
+        default_emergency_admin,
+        default_ownership_admin,
+        alice,
+    ]:
+        with brownie.reverts():
+            root_gauge_factory_proxy.set_killed(root_gauge, True, {"from": unauthorised_admins})
+
+    # so transfer root gauge factory ownership to proxy:
+    root_gauge_factory.commit_transfer_ownership(root_gauge_factory_proxy, {"from": alice})
+    root_gauge_factory_proxy.accept_transfer_ownership(root_gauge_factory, {"from": bob})
+
+    # only proxy admins can kill a gauge:
+    for authorised_admin in [default_emergency_admin, default_ownership_admin]:
+        root_gauge_factory_proxy.set_killed(root_gauge, True, {"from": authorised_admin})
+        assert root_gauge.is_killed()
+        if root_gauge == root_gauge:
+            assert root_gauge.inflation_params()[0] == 0  # inflation rate of root gauge should be 0
+        chain.undo()
+
+    # unauthorised accounts cannot do so:
+    for unauthorised_acct in [alice, bob, charlie]:
+        with brownie.reverts():
+            root_gauge_factory_proxy.set_killed(root_gauge, True, {"from": unauthorised_acct})
+
+
+def test_set_bridger(root_gauge_factory, root_gauge_factory_proxy, alice, bob, chain):
```

Comment:
break into happy/sad paths as well, set bridger success and revert
One thing to note is, no need to do integration checks, i.e. verifying that the proxy can't set the bridger since it isn't the factory owner yet

Best to have the precondition that the proxy is already the owner, and verify the revert/success cases from there. Since we already know from the gauge unit tests that only the owner can kill it